org.xtend.generators/org.xtend.smartsoft.generator/src/org/xtend/smartsoft/generator/component/ActivityGeneratorExtension.xtend

```diff
@@ -134,4 +134,38 @@ interface ActivityGeneratorExtension {
 	 * @result the update code as a character stream 
 	 */
 	def CharSequence getUpdateValuesImplementation(Activity activity) ''''''
+	
+	/**
+	 * This optional method places provided sniplet into the <b>update method</b> 
```

Comment:
These comments are copy pasted from the above method definition, a meaningful comment/documentation should be provided.

Reply:
@ipa-nhg to save time I will add the comments myself, but next time please also document the extensions properly...

Reply:
Ups sorry, I was completely busy yesterday. I will check the comments next time.